### PR TITLE
library versions update

### PR DESCRIPTION
**Pull request by blublinsky**
## Why are these changes needed?
updated library/kfp image versions to 0.1.1 to capture the latest changes

### CODE REVIEW

File: .make.versions [hunk 1]
```diff
@@ -23,5 +23,5 @@ CODE_QUALITY_VERSION=0.3.0
 DOC_QUALITY_VERSION=0.3.0
 INGEST_TO_PARQUET_VERSION=0.3.0
 
-KFP_DOCKER_VERSION=0.2.0
+KFP_DOCKER_VERSION=0.1.1
```
Comment (roytman) [inline .make.versions:26]: why did you returned the major version back?

File: kfp/kfp_support_lib/pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [project]
 name = "data_prep_toolkit_kfp"
-version = "0.1.0"
+version = "0.1.1"
```
Comment (roytman) [inline kfp/kfp_support_lib/pyproject.toml:3]: the version is wrong

File: kfp/kfp_ray_components/createRayClusterComponent.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ inputs:
 
 implementation:
     container:
-        image: quay.io/dataprep1/data-prep-kit/kfp-data-processing:0.2.0
+        image: quay.io/dataprep1/data-prep-kit/kfp-data-processing:0.1.1
```
Comment (roytman) [inline kfp/kfp_ray_components/createRayClusterComponent.yaml:14]: it should be 0.2.1

**Review by roytman — CHANGES_REQUESTED** [review]

File: .make.versions (hunk 1 above)
Comment (blublinsky) [reply .make.versions:26]: because we are really on 0.1.1. Trying to be consistent

**Comment by roytman** [comment]
the pipelines should be updates as well

File: kfp/kfp_support_lib/pyproject.toml (hunk 2 above)
Comment (blublinsky) [reply kfp/kfp_support_lib/pyproject.toml:3]: no, its 0.1.1, which is correct

File: kfp/kfp_ray_components/createRayClusterComponent.yaml (hunk 3 above)
Comment (blublinsky) [reply kfp/kfp_ray_components/createRayClusterComponent.yaml:14]: See above

**Comment by blublinsky** [comment]
> he pipelines should be updates as well

done

**Review by roytman — APPROVED** [review]

**Pull request merged**